x/programs/cmd/simulator/vm/actions/program_execute.go

```diff
@@ -101,14 +101,14 @@ func (t *ProgramExecute) Execute(
 	importsBuilder.Register("state", func() host.Import {
 		return pstate.New(logging.NoLog{}, mu)
 	})
-	callContext := program.Context{
+	callContext := &program.Context{
 		ProgramID: programID,
 		// Actor:            [32]byte(actor[1:]),
 		// OriginatingActor: [32]byte(actor[1:])
 	}
 
 	importsBuilder.Register("program", func() host.Import {
-		return importProgram.New(logging.NoLog{}, eng, mu, cfg, &callContext)
+		return importProgram.New(logging.NoLog{}, eng, mu, cfg, callContext)
```

Comment:
Is there any particular reason why we don't pass `t.Log` here (and above) ? Logs cannot be printed to the terminal otherwise. I know that they are broken currently but https://github.com/ava-labs/hypersdk/pull/884 fixes this as a side-effect.